### DISCUSSION TITLE
Open3DML training pipeline doesnt find labels -- where do I config this?

### DISCUSSION THREAD

**Discussion by runra** [isl-org/Open3D#4919] [Q&A]
HI All ML Gurus, 

**The crux of the question: How do I structure my data set so this code finds the training labels and the images?** 

I'm trying to use some of the example code to test out training with the semantickitti data set.
I've downloaded it, and got semantic segmentation to work. 

However, when I try to run the following code, it cannot find the labels...(FileNotFoundError:  Label file)
`import open3d.ml as _ml3d
import open3d.ml.torch as ml3d

cfg_file = "/home/uname/PycharmProjects/Open3DML-Clean3/Open3D-ML-master/ml3d/configs/randlanet_semantickitti.yml"
cfg = _ml3d.utils.Config.load_from_file(cfg_file)

#model = ml3d.models.RandLANet(**cfg.model)
cfg.dataset['dataset_path'] = "/media/uname/examplePCD/data_odometry_velodyne/" 
# use a cache for storing the results of the preprocessing (default path is './logs/cache')
dataset = ml3d.datasets.SemanticKITTI(dataset_path="/media/uname/examplePCD/data_odometry", use_cache=True)

# create the model with random initialization.
#model = RandLANet()
model = ml3d.models.RandLANet()

pipeline = ml3d.pipelines.SemanticSegmentation(model=model, dataset=dataset, max_epoch=100)

# prints training progress in the console.
pipeline.run_train()`

Now...the labels are in:
/media/uname/examplePCD/data_odometry_labels

How do I configure the pipeline and the above script to find both the labels and the training data?

Thanks for your help.

**Comment by Rexus-Reborn** [comment]
Hey , I want to know what config you used to run the open3d model, I am unable to find the correct version on windows 10
